wcfsetup/install/files/lib/system/session/SessionHandler.class.php

```diff
@@ -991,4 +984,140 @@ protected function getSpiderID($userAgent) {
 	public function isFirstVisit() {
 		return $this->firstVisit;
 	}
+	
+	/**
+	 * Returns all user sessions for a specific user.
+	 *
+	 * @return      \wcf\system\session\Session[]
+	 * @since       5.4
+	 */
+	public function getUserSessions(User $user): array {
+		if ($user->userID === 0) {
```

Comment:
```suggestion
		if (!$user->userID) {
```

To handle both `0` and `null`.